.azure/azure-nightly-template-windows.yml

```diff
@@ -43,7 +43,7 @@ jobs:
   - powershell: |
       $env:STACK_ROOT = "$(Build.SourcesDirectory)\.stack-root"
       $env:PATH += ";$env:HOME\.local\bin"
-      stack etc/scripts/release.hs --arch=i386 build
+      stack etc/scripts/release.hs --arch=x86_64 build
```

Comment:
This MR looks like it has the same issue as #5189 -- builds 64-bit release twice.